controller/workitem_blackbox_test.go

```diff
@@ -674,11 +681,18 @@ func createOneRandomIteration(ctx context.Context, db *gorm.DB) *iteration.Itera
 }
 
 func createOneRandomArea(ctx context.Context, db *gorm.DB, testName string) *area.Area {
+	spaceTemplate, err := spacetemplate.CreateEmptySpaceTemplate(ctx, db)
+	if err != nil {
+		fmt.Println("failed to create space template")
```

Comment:
It is used all over the place in this test file but I've fixed it in bd3c077eb526b085274a7b55a44c1ed5cf766239